.cargo/audit.toml

```diff
@@ -0,0 +1,5 @@
+[advisories]
+ignore = [
+    "RUSTSEC-2020-0071", # `chrono` is a transient dependency used by `near-sandbox-utils` and other dependencies
```

Comment:
So it seems that we cannot upgrade it, but we also don't hit it, right?

Reply:
I mean, we could override the dependency if we wanted to, but that isn't the best solution. Or otherwise upgrade all the deps we control to a newer version.